frontend/lib/about/texts.dart

```diff
@@ -12,12 +12,7 @@ List<Widget> getCopyrightText(BuildContext context) {
   <p>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.</p>
   <p>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.</p>
   ''';
-  return [
-    Html(
-      data: content,
-      style: _getHtmlStyle(),
-    ),
-  ];
+  return [Html(data: content, style: _getHtmlStyle())];
```

Comment:
Some parts got formatted perfectly and some like here is compact

Reply:
i just ran the command